gwvolman/lib/dataone/publish.py

```diff
@@ -76,6 +77,11 @@ def publish(self, tale_id, gc, dataone_node, dataone_auth_token,
             # We'll want to exit if we can't create the client
             raise ValueError('Failed to establish connection with DataONE.')
 
+        # Set the coordinating node
+        coordinating_node_url = D1_ENV_DICT['dev']['base_url']
+        if is_production:
+            D1_ENV_DICT['prod']['base_url']
```

Comment:
I think you meant `coordinating_node_url = D1_ENV_DICT['prod']['base_url']` here

Reply:
My idea was to default the value to the development server, and set it to production if `is_production` is true. Is defaulting like that a bad practice?

Reply:
It's a good practice. However, forgetting left hand side of the assignment is slightly worse ;)

Reply:
Oh wait it took me two times to see that :/

I'm probably going to pass the CN URL to gwvolman since we'll be taking a key in the girder_wholetale endpoint.